src/solve.C

```diff
@@ -332,6 +332,9 @@ void EW::solve(vector<Source*>& a_Sources, vector<TimeSeries*>& a_TimeSeries,
   vector<int> identsources;
   sort_grid_point_sources(point_sources, identsources);
 
+  // Potentially disable restart if no checkpoint is found
+  m_check_point->verify_restart();
+
```

Comment:
@Ram3sh , I just realized this probably needs to go just before the call to ``getDt()``, since that's the first place where the SCR restart calls are.  Or it could go right after something like ``parseInput``.